src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -198,6 +253,118 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
         }
     }
 
+    /**
+     * Try to spawn a new citizen as child.
+     */
+    public void trySpawnChild()
+    {
+        // Spawn a child when adults are present and the house has space, so level 3+
+        if (colony.canMoveIn() && femalePresent && malePresent && colony.getCitizenManager().getCurrentCitizenCount() < colony.getCitizenManager().getMaxCitizens())
+        {
+            CitizenData mom = null;
+            CitizenData dad = null;
+
+            for (final CitizenData data : getAssignedCitizen())
```

Comment:
I like that you made this depend on the mom and the. However i would like to see improvements here in the future. Possibly marriage or other elements should be required to boost the children chance.

This would allow the player to control who becomes a child of whom, giving him the control of who actually gets children and when.

As i see it now this kinda works like a marriage, since it always picks the first and breaks the loop.
This should however be controlled by, or at least communicated to, the player. @Wissi could you see to it, that this comes in the Wiki somewhere clearly?

Reply:
Yeah, we want to increase this system later with elements like marriage and schools etc. Marriage might come when we got the religion system in.

Reply:
Yeah, for now i just want clearly documented how the new system can be controlled.

Reply:
ye wasnt sure how detailed/micromanagement heavy it should become, since the players already got a few things to do by hand